### ISSUE TITLE
Policy to check for publicly writeable wiki

### ISSUE THREAD

**Issue by artis3n** [ossf/allstar#321]
Allstar should have a check for whether a repo has configured their wiki to be publicly writeable (and therefore open to defacement, manipulation).

**Comment by jeffmendoza** [comment]
Yes, this would be good, but there is currently not an API for it I believe.

**Comment by artis3n** [comment]
I think you're right - you could script some manual browsing but I think that's going to have to remain unfulfilled unless GitHub exposes an API.

**Comment by jeffmendoza** [comment]
Yeah, some GitHub automation tools resort to browser emulators. Not sure I want to go down that route. We would probably need a robot user to accompany the app, as the app can't auth to the website.

**Comment by Gby56** [comment]
Opened a discussion thread for Github product feedback https://github.com/orgs/community/discussions/51582

**Comment by justaugustus** [comment]
We're not going to explore this, as there does not appear to be a way to configure this via GitHub's APIs.
